CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# Release 2.36.0 25-04-2022
+
+- Upgraded Envoy to 1.22.0 (#684)
+- Added option to enable strict request validation in transcoder (#685)
```

Comment:
remove line 4, as it is not exposed to ESPv2 users.

Reply:
Do you mean remove this line "Added option to enable strict request validation in transcoder"?

Reply:
Removed